app/screens/channel/channel_nav_bar/channel_title/channel_title.js

```diff
@@ -82,9 +112,11 @@ export default class ChannelTitle extends PureComponent {
                     >
                         {channelName}
                     </Text>
+                    {guestBadge}
                     {icon}
                     {mutedIcon}
                 </View>
+                {hasGuestsText}
```

Comment:
adding a second row to the channel title won't work on landscape

Reply:
Now this is not shown when you are in mobile and landscape (shown in portrait and landscape on tablets).